setup.py

```diff
@@ -57,6 +57,7 @@
             "black>=22.1.0",
             "flake8>=4.0.1",
             "graphviz>=0.10.1",
+            "gym>=0.26.1",
```

Comment:
Gym has been quite unstable for the last couple of months. We should freeze the version for the following month to wait for the gym team to stabilize the API.

Reply:
Sure! What version do you suggest?

Reply:
Let's cap and not freeze then @AdilZouitine 👍

Reply:
Yes I think is a good option!